pkg/ddl/cluster.go

```diff
@@ -349,10 +349,52 @@ func GetTableDataKeyRanges(nonFlashbackTableIDs []int64) []kv.KeyRange {
 	return keyRanges
 }
 
-// GetFlashbackKeyRanges get keyRanges for flashback cluster.
+type keyRangeMayExclude struct {
+	r       kv.KeyRange
+	exclude bool
+}
+
+// appendContinuousKeyRanges merges not exclude continuous key ranges and appends
+// to given []kv.KeyRange, assuming the gap between key ranges has no data.
+//
+// Precondition: schemaKeyRanges is sorted by start key. schemaKeyRanges are
+// non-overlapping.
+func appendContinuousKeyRanges(result []kv.KeyRange, schemaKeyRanges []keyRangeMayExclude) []kv.KeyRange {
```

Comment:
```suggestion
func mergeContinuousKeyRanges(result []kv.KeyRange, schemaKeyRanges []keyRangeMayExclude) []kv.KeyRange {
```

Reply:
I want to mention the input slice will be `append`ed, so I prefer keep the old name

Reply:
maybe avoid modifying input param, ok to leave it anyway